packages/connect/src/connector/specs/index.ts

```diff
@@ -0,0 +1,10 @@
+export async function getSpec(chain: string): Promise<string> {
+  const specRaw = (await import("./generated/" + chain + ".js")) as
+    | string
+    | { default: string }
+
+  return typeof specRaw === "string"
+    ? /* istanbul ignore next line */
```

Comment:
What's istanbul?

Reply:
I just removed it. It lets the test framework know that line is not to be taken into account for the code-coverage.